### PR TITLE
Update graph-edge-constraints.md

### PR DESCRIPTION
**Pull request by JerryNixon**
The description of how multiple edge constraints and multiple edge constraint clauses operate under-the-cover was addressed to be clearer to the developer.

### CODE REVIEW

**Comment by PRMerger18** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by arvindshmicrosoft** [comment]
@JerryNixon thank you for the contribution! I will review ASAP. @MikeRayMSFT - would you please assign this to me?

File: docs/relational-databases/tables/graph-edge-constraints.md [hunk 1]
```diff
@@ -32,13 +32,40 @@ SQL Graph supports edge constraints, which enable users to add constraints to th
...
+```
+CONSTRAINT EC_BOUGHT CONNECTION (Supplier TO Product, Customer TO Product)
+``` 
+In this case, one edge constraint specifies two edge constraint clauses. The constraint clauses allow the `bought` edge to contain BOTH `Supplier -(bought)-> Product` and `Customer -(bought)-> Product` relationships. Inserting any other node pairs would fail. 
+
+```
+CONSTRAINT EC_BOUGHT1 CONNECTION (Supplier TO Product)
+CONSTRAINT EC_BOUGHT2 CONNECTION (Customer TO Product)
+``` 
+In this case, two constraints specify one constraint clause each. These only allow inserts that satisfy BOTH edge constraint clauses simultaneously. This is, of course, impossible. There is no node pair that can satify both edge constraint clauses. This edge constraint combination makes the edge table unusable.
```
Comment (arvindshmicrosoft) [inline docs/relational-databases/tables/graph-edge-constraints.md:68]:
```suggestion
The above example shows two constraints on the same edge table, with each edge constraint specifying one constraint clause. In this situation, SQL would only allow inserts that satisfy BOTH edge constraint clauses simultaneously. This is, of course, impossible. There is no node pair that can satisfy both edge constraint clauses. This edge constraint combination makes the edge table unusable.

For a detailed explanation of where multiple edge constraints can be used in a real-life scenario, please see the example "Creating a new edge constraint on existing edge table, with new edge constraint clause" later in this page.
```

**Review by arvindshmicrosoft — CHANGES_REQUESTED** [review]
Thank you @JerryNixon for your valuable contribution! Your changes certainly simplify this complex topic. I have reviewed and suggested a few changes. If you can please push up a commit to address these, we can merge the contribution.

**Comment by PRMerger20** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@JerryNixon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by arvindshmicrosoft — APPROVED** [review]
#sign-off

**Comment by MikeRayMSFT** [comment]
#hold-off

I'm going to pull this into the private repository to validate in a preview-staging environment.

**Pull request merged**